src/atsamd/Kconfig

```diff
@@ -162,7 +173,7 @@ endchoice
 
 config CLOCK_FREQ
     int
-    default 48000000 if MACH_SAMD21
+    default 48000000 if MACH_SAMC21 || MACH_SAMD21
```

Comment:
I think it would be a good idea to add a new `MACH_SAMX2` definition (similar to the existing `MACH_SAMX5`) to reduce the number of cases of `CONFIG_MACH_SAMC21 || CONFIG_MACH_SAMD21`.